.github/workflows/ghes-acceptance-tests-all.yml

```diff
@@ -0,0 +1,39 @@
+name: GHES Acceptance Tests (All)
+
+on:
+  schedule:
+    #         ┌───────────── minute (0 - 59)
+    #         │ ┌───────────── hour (0 - 23)
+    #         │ │ ┌───────────── day of the month (1 - 31)
+    #         │ │ │ ┌───────────── month (1 - 12 or JAN-DEC)
+    #         │ │ │ │ ┌───────────── day of the week (0 - 6 or SUN-SAT)
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         │ │ │ │ │
+    #         * * * * *
+    - cron:  '*/15 * * * *'
+
+jobs:
+  acceptance-tests:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 2
+
+      - name: Acceptance Tests
+        uses: terraformtesting/acceptance-tests@v2.0.0
+        with:
+          RUN_ALL: true
+          GITHUB_BASE_URL: "https://terraformtesting-ghe.eastus.cloudapp.azure.com/"
```

Comment:
Just noticed that this GHE endpoint uses a self-signed certificate. My experience with GHE is quite outdated now, but doesn't it support Let's Encrypt? Either way I am surprised that this doesn't require the insecure flag 🤔

Reply:
Yes, this will absolutely fail.  I have another PR behind this one to address correctness.  This PR lays a foundation from an Actions configuration perspective, but intends to be iterated on afterwards.